service/src/main/java/greencity/service/UserServiceImpl.java

```diff
@@ -1134,8 +1134,6 @@ public PageableDto<UserAllFriendsDto> findFriendByName(String name, Pageable pag
             new TypeToken<List<UserAllFriendsDto>>() {
             }.getType());
 
-        friendDtos.stream().forEach(f -> f.setFriendsChatDto(restClient.chatBetweenTwo(f.getId(), id)));
-
```

Comment:
Why did you remove it?
Should it also be removed from `findAllUsersFriends` method?